blueprints/controllers/ikea_e1812/ikea_e1812.yaml

```
@@ -4,7 +4,8 @@ blueprint:
   description: |
     # Controller - IKEA E1812 TRÅDFRI Shortcut button
 
-    Controller automation for executing any kind of action triggered by the provided IKEA E1812 TRÅDFRI Shortcut button. Allows to optionally loop an action on a button long press.
+    Controller automation for executing any kind of action triggered by the provided IKEA E1812 TRÅDFRI Shortcut button. 
```

```suggestion
    Controller automation for executing any kind of action triggered by the provided IKEA E1812 TRÅDFRI Shortcut button. Allows to optionally loop an action on a button long press.
```
Keeping it as it was before the update.